pkgs/development/python-modules/setuptools-scm-git-archive/default.nix

```diff
@@ -0,0 +1,22 @@
+{ stdenv, buildPythonPackage, fetchPypi, setuptools_scm, pytest }:
+
+buildPythonPackage rec {
+  pname = "setuptools_scm_git_archive";
```

Comment:
Can we have `pname` use dashes too?

Reply:
Ideally it is also the normalized name, however, then likely fetchPypi needs a different value

Reply:
I've updated the pname to the pypi name, and updated fetchPypi accordingly